prep-html-syntax/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <h1 style="color:green;">My favorite Star Wars character is Yoda</h1>
+  <img src="https://media.istockphoto.com/photos/master-yoda-picture-id501599698?k=20&m=501599698&s=612x612&w=0&h=4YQxqUkzHj4Ucbqhuc9VqM6l6nf0ADD7cKMAB_2pJ0s=" alt="yoda">
+  <p style="color:blue;">I have never actually watched Star Wars but find Yoda super cute</p>
+  </html>
+</html>
```

Comment:
Hi Mary, looks like you have an extra closing tag for your `<html></html>` element. Also, be sure to put the visible content of your document in a `<body></body>` element